packages/metablocks/src/accounts/universe.ts

```diff
@@ -0,0 +1,36 @@
+import * as anchor from "@project-serum/anchor";
+import { MetaBlocks } from "../types/meta_blocks";
+import { Universe } from "../types/types";
+import { camelToSnakeCaseArrayObject, setBlockMetadata } from "./utils";
+
+const getAllUniverses = async (
+  program: anchor.Program<MetaBlocks>
+): Promise<Universe[]> => {
+  const universeAccounts = await program.account.universe.all();
+
+  let universes: Array<Universe> = [];
+
+  for (const element of universeAccounts) {
```

Comment:
Can we do a map here ? 
```js
 universeAccounts.map(element => {
   const universeAccount = element.account;
   const metadata = await setBlockMetadata(element.publicKey, program);

    return {
      publicKey: element.publicKey.toString(),
      authority: universeAccount.universeAuthority.toString(),
      name: universeAccount.config.name as string,
      websiteUrl: universeAccount.config.websiteUrl as string,
      description: universeAccount.config.description as string,
      lastUpdateTs: universeAccount.lastUpdateTs.toNumber(),
      totalNfts: universeAccount.totalNfts.toNumber(),
      blockTime: metadata.blockTime,
      signature: metadata.signature,
      slot: metadata.slot,
    };
})
```

`push` is mutable. You can say no to this change, I'm trying to influence you because I feel that functional constructs are superior.